### DISCUSSION TITLE
Is the versioning spec available anywhere?

### DISCUSSION THREAD

**Discussion by mybearworld** [TodePond/GulfOfMexico#507] [Q&A]
I am impressed with the versioning system of nDreamBerd. Is it available for use anywhere?

<table><tr><td>v0.0.0.0.0.0.0.0.0</td><td>v0.0.0.0.0.0.0.0.0a</td><td>v-0.0.0.0.0.0.0.0.1</td><td>1.0.0.0.0.0.0.0.0</td><td>v1.0.0.0.0.0.0.0.0a</td><td>version.one.zero.one</td><td>v0.9.9.9.9.9.9.9.9</td><td>v0.9.9.9.9.9.9.9.9a</td><td>v0.9.9.9.9.9.9.9.9b</td><td>v0.9.9.9.9.9.9.9.9c</td><td>v0.9.9.9.9.9.9.9.9d</td><td>v.release.notes</td><td>v0.9c</td><td>v0.9cllm</td><td>v0.9wwpsmdfosajluaahniavw</td><td>vyesterday</td><td>v0.9z</td><td>v2.0</td><td>v3.0</td><td>v$.$</td><td>v181.81</td><td>v460.18</td><td>v0.0</td><td>vNaN.NaN</td><td>vaN.aN</td><td>v1.0</td><td>vv1.0</td><td>vvv1.0</td><td>...</td></tr></table>

**Comment by Forest-Terminal** [comment]
No I do not think so as the compiler and compiler compiler still need to be made fully opperational

**Comment by circle-gon** [comment]
I was just going to check out the LICENSE.md to see if it could be usable by others and I stumbled across the example page. Thanks!

More seriously: you could probably use it, I don't think TodePond would be mad about it.
